### PR TITLE
cachedir should be /cloud not /master

### PR DESCRIPTION
**Pull request by gtmanfred**
### What does this PR do?

Fix regression that moved the cloud cachedir to /master
### Tests written?

Not yet

### CODE REVIEW

**Comment by gtmanfred** [comment]
@techhat can you review this?

**Comment by vutny** [comment]
I've also submitted an alternative fix: #35898 
Could you please take a look?

File: salt/utils/cloud.py [hunk 1]
```diff
@@ -2608,7 +2608,7 @@ def list_cache_nodes_full(opts, provider=None, base=None):
         return
 
     if base is None:
-        base = os.path.join(__opts__['cachedir'], 'cloud', 'active')
+        base = __opts__['cachedir']
```
Comment (vutny) [inline salt/utils/cloud.py]: Maybe missed `active` subdir here?

**Comment by techhat** [comment]
+1

**Pull request merged**